{"schemaVersion":{"patch":0,"major":0,"minor":3},"kind":"symbol","identifier":{"interfaceLanguage":"swift","url":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/CredentialsManagerError\/noCredentials"},"sections":[],"abstract":[{"text":"No credentials were found in the store.","type":"text"},{"type":"text","text":" "},{"type":"text","text":"This error does not include a "},{"identifier":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/Auth0Error\/cause-9wuyi","type":"reference","isActive":true},{"text":".","type":"text"}],"hierarchy":{"paths":[["doc:\/\/com.auth0.Auth0\/documentation\/Auth0","doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/CredentialsManagerError"]]},"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/auth0\/credentialsmanagererror\/nocredentials"]}],"metadata":{"roleHeading":"Type Property","externalID":"s:5Auth023CredentialsManagerErrorV02noB0ACvpZ","title":"noCredentials","role":"symbol","modules":[{"name":"Auth0"}],"symbolKind":"property","fragments":[{"text":"static","kind":"keyword"},{"kind":"text","text":" "},{"text":"let","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"noCredentials"},{"kind":"text","text":": "},{"text":"CredentialsManagerError","kind":"typeIdentifier","preciseIdentifier":"s:5Auth023CredentialsManagerErrorV"}]},"primaryContentSections":[{"declarations":[{"languages":["swift"],"platforms":["iOS"],"tokens":[{"text":"static","kind":"keyword"},{"kind":"text","text":" "},{"text":"let","kind":"keyword"},{"kind":"text","text":" "},{"text":"noCredentials","kind":"identifier"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"CredentialsManagerError","identifier":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/CredentialsManagerError","preciseIdentifier":"s:5Auth023CredentialsManagerErrorV"}]}],"kind":"declarations"}],"references":{"doc://com.auth0.Auth0/documentation/Auth0/Auth0Error/cause-9wuyi":{"type":"topic","identifier":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/Auth0Error\/cause-9wuyi","title":"cause","kind":"symbol","role":"symbol","fragments":[{"text":"var","kind":"keyword"},{"kind":"text","text":" "},{"text":"cause","kind":"identifier"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"Error","preciseIdentifier":"s:s5ErrorP"},{"text":"?","kind":"text"}],"url":"\/documentation\/auth0\/auth0error\/cause-9wuyi","abstract":[{"text":"The underlying ","type":"text"},{"code":"Error","type":"codeVoice"},{"text":" value, if any. Defaults to ","type":"text"},{"type":"codeVoice","code":"nil"},{"text":".","type":"text"}]},"doc://com.auth0.Auth0/documentation/Auth0":{"url":"\/documentation\/auth0","type":"topic","identifier":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0","title":"Auth0","abstract":[{"type":"text","text":"SDK for Apple platforms."}],"kind":"symbol","role":"collection"},"doc://com.auth0.Auth0/documentation/Auth0/CredentialsManagerError/noCredentials":{"fragments":[{"text":"static","kind":"keyword"},{"kind":"text","text":" "},{"text":"let","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"noCredentials"},{"kind":"text","text":": "},{"kind":"typeIdentifier","preciseIdentifier":"s:5Auth023CredentialsManagerErrorV","text":"CredentialsManagerError"}],"abstract":[{"text":"No credentials were found in the store.","type":"text"},{"type":"text","text":" "},{"text":"This error does not include a ","type":"text"},{"type":"reference","identifier":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/Auth0Error\/cause-9wuyi","isActive":true},{"text":".","type":"text"}],"type":"topic","title":"noCredentials","kind":"symbol","role":"symbol","url":"\/documentation\/auth0\/credentialsmanagererror\/nocredentials","identifier":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/CredentialsManagerError\/noCredentials"},"doc://com.auth0.Auth0/documentation/Auth0/CredentialsManagerError":{"type":"topic","fragments":[{"kind":"keyword","text":"struct"},{"kind":"text","text":" "},{"kind":"identifier","text":"CredentialsManagerError"}],"kind":"symbol","role":"symbol","title":"CredentialsManagerError","abstract":[{"text":"Represents an error during a Credentials Manager operation.","type":"text"}],"navigatorTitle":[{"kind":"identifier","text":"CredentialsManagerError"}],"url":"\/documentation\/auth0\/credentialsmanagererror","identifier":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/CredentialsManagerError"}}}